go/cmd/vtctldclient/command/vreplication/vdiff/vdiff.go

```diff
@@ -60,6 +60,7 @@ var (
 		Limit                       uint32 // We only accept positive values but pass on an int64
 		FilteredReplicationWaitTime time.Duration
 		DebugQuery                  bool
+		MaxReportSampleRows         uint64
```

Comment:
We should make this a uint32 as the internal variable is an int64 but we want the positive value requirements for the user provided value.

Reply:
fixed!